sql/src/main/java/io/crate/executor/transport/task/elasticsearch/ESGetTask.java

```diff
@@ -122,10 +125,10 @@ public ESGetTask(UUID jobId,
             SettableFuture<Bucket> settableFuture = SettableFuture.create();
             listener = new GetResponseListener(settableFuture, extractors);
             results = ImmutableList.of(Futures.transform(settableFuture, QueryResult.TO_TASK_RESULT));
+            context = new ESJobContext("lookup by primary key", ImmutableList.of(request), ImmutableList.of(listener), results, transportAction);
```

Comment:
ah yeah, didn't see this. but cant you define projectorChain outside and start it later if its not null?